site/content/how-to/monitoring/troubleshooting.md

```diff
@@ -65,54 +79,312 @@ LAST SEEN   TYPE      REASON              OBJECT
+
+```shell
+kubectl describe pod <ngf-pod-name> -n nginx-gateway
+```
+
+The Pod description includes details about the image name, tags, current status, and environment variables. Verify that these details match your setup and cross-check with the events to ensure everything is functioning as expected. For example, the Pod below has two containers that are running and the events reflect the same.
+
+```text
+Containers:
+  nginx-gateway:
+    Container ID:  containerd://06c97a9de938b35049b7c63e251418395aef65dd1ff996119362212708b79cab
+    Image:         nginx-gateway-fabric
+    Image ID:      docker.io/library/import-2024-06-13@sha256:1460d63bd8a352a6e455884d7ebf51ce9c92c512cb43b13e44a1c3e3e6a08918
+    Ports:         9113/TCP, 8081/TCP
+    Host Ports:    0/TCP, 0/TCP
+    State:          Running
+      Started:      Thu, 13 Jun 2024 11:47:46 -0600
+    Ready:          True
+    Restart Count:  0
+    Readiness:      http-get http://:health/readyz delay=3s timeout=1s period=1s #success=1 #failure=3
+    Environment:
+      POD_IP:          (v1:status.podIP)
+      POD_NAMESPACE:  nginx-gateway (v1:metadata.namespace)
+      POD_NAME:       ngf-nginx-gateway-fabric-66dd665756-zh7d7 (v1:metadata.name)
+  nginx:
+    Container ID:   containerd://c2f3684fd8922e4fac7d5707ab4eb5f49b1f76a48893852c9a812cd6dbaa2f55
+    Image:          nginx-gateway-fabric/nginx
+    Image ID:       docker.io/library/import-2024-06-13@sha256:c9a02cb5665c6218373f8f65fc2c730f018d0ca652ae827cc913a7c6e9db6f45
+    Ports:          80/TCP, 443/TCP
+    Host Ports:     0/TCP, 0/TCP
+    State:          Running
+      Started:      Thu, 13 Jun 2024 11:47:46 -0600
+    Ready:          True
+    Restart Count:  0
+    Environment:    <none>
+Events:
+  Type    Reason     Age   From               Message
+  ----    ------     ----  ----               -------
+  Normal  Scheduled  40s   default-scheduler  Successfully assigned nginx-gateway/ngf-nginx-gateway-fabric-66dd665756-zh7d7 to kind-control-plane
+  Normal  Pulled     40s   kubelet            Container image "nginx-gateway-fabric" already present on machine
+  Normal  Created    40s   kubelet            Created container nginx-gateway
+  Normal  Started    39s   kubelet            Started container nginx-gateway
+  Normal  Pulled     39s   kubelet            Container image "nginx-gateway-fabric/nginx" already present on machine
+  Normal  Created    39s   kubelet            Created container nginx
+  Normal  Started    39s   kubelet            Started container nginx
+```
+
+
+### Modify logging levels
+
+To debug NGINX Gateway Fabric, enable verbose logging by editing the `NginxGateway` configuration. This can be done either before or after deploying NGINX Gateway Fabric. Refer to this [guide](https://docs.nginx.com/nginx-gateway-fabric/how-to/configuration/control-plane-configuration) to do so.
```

Comment:
I would be clear that this ONLY pertains to the control plane logs, not the nginx logs.